docs/concepts/redirects.mdx

```diff
@@ -3,35 +3,33 @@ id: redirects
 title: Browser Redirects and Flow Completion
 ---
 
-Ory Cloud offers methods to integrate your own user interface (UI) such as
-utilizing the SDK in Single Page Applications (ReactJs, Angular, VueJs etc.) or
-using redirects in Server Side Applications (NodeJs, PHP, Golang etc.).
-
-On this page of the documentation we will be covering Browser Redirects and how
-to configure it on Ory Cloud console. If you are interested in utilizing the SDK
-for Single Page Applications, check out our [SDK](./services-api.mdx)
-documentation.
-
-For a deeper dive into the background of this setup, head over to our open
-source documentation:
-
-- https://www.ory.sh/kratos/docs/concepts/browser-redirect-flow-completion
-
-## Allow list
-
-Ory Cloud allows dynamic redirects to be set using the `?return_to=` query
-parameter on [Self-Service Flows](./self-service.mdx). As an example a user
-opens a sharable link to go to https://myapp.com/posts, however, this url
-requires the user to be signed in and thus redirects the user back to the login
-page. To return the user back to the original url we can append
-`?return_to=https://myapp.com/posts` to the specified Self-Service initialise
-flow url.
-
-The `Allow list` is used in this scenario to prevent Open Redirect Attacks by
-only allowing certain domains, or paths on a domain. As shown in this example
-below the https://sub.domain.myapp.com/only/path needs to match perfectly for
-this sub-domain and path for it to be allowed. Other redirects using myapp.com
-will fail, however, for https://anotherapp.com will succeed on any path.
+This document covers browser redirects for Server Side Applications (NodeJs,
+PHP, Golang etc.) and how to configure them. To learn more about the SDK for
+Single Page Applications, check out the [SDK](./services-api.mdx) documentation.
+
+## Allow List
+
+Set dynamic redirects using the `?return_to=` query parameter on
+[Self-Service Flows](./self-service.mdx). For example: a user opens a sharable
+link to go to `https://myapp.com/posts`. This URL requires the user to have an
+active session and redirects the user back to the login page. To return the user
+back to the original URL, append `?return_to=https://myapp.com/posts` when
+starting the self-service login flow `initSelfServiceLogin`:
+
+```js
+    "request_url": "http://<your-project>.projects.oryapis.com/self-service/login/browser?return_to=...",
```

Comment:
This is confusing, `initSelfServiceLogin`  does not accept the aprameter like this. It also depends on the SDK that you're using. Maybe it makes more sense to link to the API doc here:

https://www.ory.sh/docs/reference/api#operation/submitSelfServiceLoginFlow

and show an example with cURL:

```suggestion
curl -X GET 'http://<your-project>.projects.oryapis.com/self-service/login/browser?return_to=...'
```